<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tblPanen">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>621</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID Panen</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID karyawan</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tanggal Panen</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hasil Panen</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total Panen</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>491</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDPanenLabel">
      <property name="text">
       <string>ID Panen</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editID"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaKaryawanLabel">
      <property name="text">
       <string>Nama Karyawan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cmbKaryawan"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tanggalPanenLabel">
      <property name="text">
       <string>Tanggal Panen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="datePanen"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="hasilPanenLabel">
      <property name="text">
       <string>Hasil Panen (Kg)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="sbHasil"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="totalPanenRpLabel">
      <property name="text">
       <string>Total Panen (Rp)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="editTotal">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>KELOLA DATA PANEN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSimpan">
      <property name="text">
       <string>Simpan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUbah">
      <property name="text">
       <string>Ubah</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHapus">
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineCari">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>501</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
